components/button/button-icon.js

```diff
@@ -49,6 +49,8 @@ class ButtonIcon extends ThemeMixin(ButtonMixin(VisibleOnAncestorMixin(RtlMixin(
 					--d2l-button-icon-min-height: calc(2rem + 2px);
 					--d2l-button-icon-min-width: calc(2rem + 2px);
 					--d2l-button-icon-h-align: calc(((2rem + 2px - 0.9rem) / 2) * -1);
+					--d2l-button-icon-text-color: var(--d2l-color-ferrite);
+					--d2l-button-icon-text-color-hover: var(--d2l-color-ferrite);
```

Comment:
Hmm... rethinking this slightly since button-icon doesn't have "text" per se (my bad for suggesting that name). Looking through this, there's also that `--d2l-icon-fill-color` variable that defaults to ferrite that we're already overriding for dark mode and the translucent style. Could we leverage it maybe?

Something like:

```css
:host {
  --d2l-icon-fill-color: var(--d2l-button-icon-color, var(--d2l-color-ferrite));
}
/* same as existing rules -- i.e. you can't override the icon colour if it's a translucent or dark mode button-icon */
:host([translucent]) {
  --d2l-icon-fill-color: white;
}
:host([theme="dark"]) {
  --d2l-icon-fill-color: var(--d2l-color-sylvite);
}
/* I'm not sure if this part will work... changing a variable value in a pseudo-class */
button:hover,
button:focus,
:host([active]) button {
  background-color: var(--d2l-button-icon-background-color-hover);
  --d2l-icon-fill-color: var(--d2l-button-icon-color-hover, var(--d2l-color-ferrite));
}
```